<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileExistsWidget</class>
 <widget class="QWidget" name="FileExistsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>137</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Folder/File Exists</string>
  </property>
  <property name="toolTip">
   <string>Folder/File Exists</string>
  </property>
  <property name="statusTip">
   <string>Folder/File Exists</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblFileName">
       <property name="text">
        <string>File or Folder Name</string>
       </property>
       <property name="buddy">
        <cstring>editFileName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkUseMacros">
       <property name="toolTip">
        <string>Translate %DD%, %MM%, %YY%, etc. as macros</string>
       </property>
       <property name="statusTip">
        <string>Translate %DD%, %MM%, %YY%, etc. as macros</string>
       </property>
       <property name="text">
        <string>Translate macros</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLineEdit" name="editFileName"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnFileName">
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grpFileTestModeRadio">
     <property name="title">
      <string> Alert when : </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rbFileDoesntExists">
          <property name="text">
           <string>file doesn't exist</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbFileExists">
          <property name="text">
           <string>file exists</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="lblAndOr">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>AND</string>
        </property>
        <property name="indent">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chkMaxAge">
          <property name="text">
           <string>older than</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkMinAge">
          <property name="text">
           <string>newer than</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QSpinBox" name="spinMaxAge">
          <property name="toolTip">
           <string>1 day = 1440 min</string>
          </property>
          <property name="statusTip">
           <string>1 day = 1440 min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinMinAge">
          <property name="toolTip">
           <string>1 day = 1440 min</string>
          </property>
          <property name="statusTip">
           <string>1 day = 1440 min</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblMinAgeMin">
          <property name="text">
           <string>min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblMaxAgeMin">
          <property name="text">
           <string>min</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
